keras_cv/layers/preprocessing/rgb_shift.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np 
+import tensorflow as tf
+from tensorflow.keras import layers, backend
+
+
+class RGBShift(layers.Layer):
+    """RGBShift class randomly shift values for each channel of the input RGB image. The expected images 
+    should be [0-255] pixel ranges.
+
+   
+    Input shape:
+        3D (unbatched) or 4D (batched) tensor with shape:
+        `(..., height, width, channels)`, in `"channels_last"` format
+    Output shape:
+        3D (unbatched) or 4D (batched) tensor with shape:
+        `(..., height, width, channels)`, in `"channels_last"` format
+
+    Args:
+        r_shift_limit: A scalar or tuple or list of two upper and lower bound number. It will change r-channel by
+            adding with the r-channel. If r_shift_limit is a single integer, the range will be (-r_shift_limit, r_shift_limit).
+            Default: (-10, 10).
```

Comment:
Btw, in https://github.com/keras-team/keras-cv/pull/122, I was using a scale value [-1.0, 1.0] as the input, which hint the limits here. It is bit unclear in this PR that what's the allowed range for those limits (I guess its -255 ~ 255)?

Reply:
Done.